kubernetes/kind-cluster-config.yml

```diff
@@ -10,6 +10,14 @@ nodes:
   extraMounts:
     - hostPath: /tmp/celestia-data
       containerPath: /data/celestia-data
+    - hostPath: /tmp/rollup-data/executor
```

Comment:
If I understand correctly, run locally all rollups will share the same directory here? Does this cause issues with starting multiple rollups?

Reply:
After working through various gross solutions, I finally found https://kubernetes.io/docs/concepts/storage/volumes/#using-subpath, which will solve this. Implementing now.